### DISCUSSION TITLE
Can we use a OpenBao Vault for testing purpuse ?

### DISCUSSION THREAD

**Discussion by AndreCbrera** [openbao/openbao#161] [Q&A] [answered]
How can we use a Vault  Dockerfile inside the project to testing purpuse? I appreciate your work thanks

**Comment by cipherboy** [comment, marked as answer]
@AndreCbrera If you run the `Makefile` via `make docker-dev` or `make docker-dev-ui`, it builds a (large) image that can be used for testing purposes. This just requires `make` and `docker`, though you can obviously run the command directly without `make`.

Hopefully the release images will be smaller. :-) HTH!

**Reply by AndreCbrera** [reply]
In my case, I don't know if the project is being able to deploy in production environment, right now we have been using a Vault by hashicorp

**Reply by joewxboy** [reply]
@AndreCbrera I work with a team that is also looking to use OpenBao in production.  We're planning to use OpenBao in production when it has a GA release, and we're planning to begin the development work of that integration beginning with the Alpha release.  I hope that's helpful to you.

**Reply by AndreCbrera** [reply]
I appreciate your answer

Thanks :)
